### PR TITLE
new guide for upcoming birthdays query in workday

### PR DESCRIPTION
**Pull request by thrilldavis**

### CODE REVIEW

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+time_in_minutes: 10
+difficulty_level: Beginner
+---
+
+# Use Case Guide: Workday: Using reporting to query for upcoming birthdays
```
Comment (ajaymerchia) [inline use-case-guides/workday-upcoming-birthdays-using-reporting/README.md:11]: I think this needs to be removed

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+In this tutorial, we will explore how to call RaaS from Creator Studio. To guide us, we will be using a pre-built report that gets the upcoming birthdays.
+
+Let's get started!
+
+# What are we building?
+
+## Conversation Design
+
+[This purple chat]() shows the experience we are going to build.
```
Comment (ajaymerchia) [inline use-case-guides/workday-upcoming-birthdays-using-reporting/README.md:27]: Missing purple chat link

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+✨ **Triggers:** Natural Language
+
+🤲 **Slots**: None
+
+🏃‍♂️ **Actions:** Retrieve data from workday report
+
+📜 **Response Data:** The data from a RaaS report
+
+📚 **Guidelines:** Existing report with no lookup fields
```
Comment (ajaymerchia) [inline use-case-guides/workday-upcoming-birthdays-using-reporting/README.md:39]: There's just ONE guideline -- "select fields to render from RaaS report" 
You can review all guidelines here: https://developer.moveworks.com/creator-studio/conversation-design/overview#guidelines

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+Based on the needs of this use case, we should build a **Lookup items with no parameters**
```
Comment (ajaymerchia) [inline use-case-guides/workday-upcoming-birthdays-using-reporting/README.md:41]: Link to design pattern?

**Review by ajaymerchia — CHANGES_REQUESTED** [review]

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    
+    ![the ellipse for the report to show custom actions](images/Untitled%203.png)
+    
+6. You will get a dialogue to sign in, press cancel and copy the URL after *[wd2-impl-services1.workday.com](http://wd2-impl-services1.workday.com)* 
+    
+    ![browser window showing url](images/Untitled%204.png)
+    
+    which should look something like this:
+    
+    <pre>/ccx/service/customreport2/[your_tenant]/[your user_name]/Alert_-_Birthdays_for_this_week_and_next</pre>
```
Comment (ajaymerchia) [inline use-case-guides/workday-upcoming-birthdays-using-reporting/README.md:85]: We don't support the "pre" HTML Tag

**Review by ajaymerchia — CHANGES_REQUESTED** [review]

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md (hunk 5 above)
Comment (thrilldavis) [reply use-case-guides/workday-upcoming-birthdays-using-reporting/README.md]: weirdest thing, had it as '''text''' but it kept throwing off the checker. Seems to be working now though

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
+---
+design_pattern_id: 5
+name: Workday - Retrieving upcoming birthdays using reporting
```
[the remark this answers is not in the document]
Comment (thrilldavis) [reply use-case-guides/workday-upcoming-birthdays-using-reporting/README.md]: shortened

File: use-case-guides/workday-upcoming-birthdays-using-reporting/README.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    
+6. You will get a dialogue to sign in, press cancel and copy the URL after *[wd2-impl-services1.workday.com](http://wd2-impl-services1.workday.com)* 
+    
+    ![browser window showing url](images/Untitled%204.png)
+    
+    which should look something like this:
+    
+    '''text
+    /ccx/service/customreport2/[your_tenant]/[your user_name]/Alert_-_Birthdays_for_this_week_and_next
+    '''
```
Comment (ajaymerchia) [inline use-case-guides/workday-upcoming-birthdays-using-reporting/README.md:87]: This needs to be done with backticks, not quotes (```)

**Review by ajaymerchia — APPROVED** [review]

**Pull request merged**